### ISSUE TITLE
Intermittent internal server errors for status db

### ISSUE THREAD

**Issue by seallard** [Clinical-Genomics/cg#2206]
## Description
When switching views in the status-db web interface, an internal server error sometimes occurs.

## This can be closed when
Internal server errors do no longer occur randomly in status-db

## Blocked by

*If there are any blocking issues/prs/things in this or other repos. Please link to them.*

**Comment by seallard** [comment]
Duplicate of https://github.com/Clinical-Genomics/cg/issues/2008
